dbft.go

```diff
@@ -76,7 +76,7 @@ func (d *DBFT[H]) Start(ts uint64) {
 	d.cache = newCache[H]()
 	d.initializeConsensus(0, ts)
 	if d.IsPrimary() {
-		d.sendPrepareRequest()
+		d.sendPrepareRequest(false)
```

Comment:
@roman-khimov this is the last thing I'd like to pay your attention to.
Old behaviour: on the first dBFT start primary will construct&relay PrepareRequest immediately.
New behaviour (with dynamic time extension enabled): on the first dBFT start primary won't construct PrepareRequest if there's no transactions in the pool. Timeout will be set to `MinTimePerBlock`. After this time subscription to mempool will be initiated and timer will be reset to `MaxTimePerBlock-MinTimePerBlock`. PrepareRequest will be sent only after this time or if there are incoming transactions.

Are we OK with this behaviour? I think we are. An alternative is `d.sendPrepareRequest(true)`.

Reply:
I doubt it changes anything significantly, but keeping the old behavior is safer, we know it worked well for a long time.